sources/web/datalab/polymer/editor.ts

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2017 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+/**
+ * This file contains functionality for the entry point of the Datalab editor. It
+ * wraps the <datalab-editor> element, and takes a "file" querystring parameter to
+ * load it in the editor.
+ */
+document.addEventListener('DOMContentLoaded', () => {
+
+  const editorElement = <DatalabEditorElement>document.querySelector('datalab-editor');
+
+  document.addEventListener('ThemeChanged', (e: CustomEvent) => {
```

Comment:
Would you suggest a common typescript file with event literals that's imported by all users? Do you have a better idea?